erpnext/projects/doctype/project_item/project_item.json

```
@@ -0,0 +1,56 @@
+{
+ "creation": "2021-07-28 01:18:44.696471",
+ "doctype": "DocType",
+ "editable_grid": 1,
+ "engine": "InnoDB",
+ "field_order": [
+  "project",
+  "status",
+  "column_break_2",
+  "time_consumed",
+  "is_default"
+ ],
+ "fields": [
+  {
+   "fieldname": "project",
+   "fieldtype": "Link",
+   "in_list_view": 1,
+   "label": "Project",
+   "options": "Project"
+  },
+  {
+   "fieldname": "status",
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "label": "Status"
+  },
+  {
+   "fieldname": "column_break_2",
+   "fieldtype": "Column Break"
+  },
+  {
+   "fieldname": "time_consumed",
+   "fieldtype": "Data",
+   "in_list_view": 1,
+   "label": "Time Consumed"
+  },
+  {
+   "default": "0",
+   "fieldname": "is_default",
+   "fieldtype": "Check",
+   "in_list_view": 1,
+   "label": "Is Default"
+  }
+ ],
+ "istable": 1,
+ "modified": "2021-07-29 23:06:05.412289",
+ "modified_by": "Administrator",
+ "module": "Projects",
+ "name": "Project Item",
```

```suggestion
 "name": "Task Project",
```

This childtable is a part of Task and not Project